### PR TITLE
19 ci pipeline 

### PR DESCRIPTION
**Pull request by Adaz99**
When you push to github it should run the tests if successful it should merge it to the branch specified (on Jenkins Project augusta-group-two-CI to check if it was successful)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Run tests
```
Comment (Charlie-robin) [inline .github/workflows/main.yml:1]: You should delete this file & folder

File: app/__tests__/scores.test.js [hunk 2]
```diff
@@ -5,90 +5,103 @@ const app = require('../app');
...
-	});
-});
+describe("testing the api", () => {
+	it('should return a status of 200 and an array containing 1 score', () => {
+		expect(true).toBeTruthy()
+	})
+})
+
+
+// test to push
```
Comment (Charlie-robin) [inline app/__tests__/scores.test.js:15]: Remove these comments

File: app/__tests__/scores.test.js [hunk 3]
```diff
@@ -5,90 +5,103 @@ const app = require('../app');
...
+//test 2
+// test 3
+// test 66555
+
+
+// beforeEach((done) => {
```
Comment (Charlie-robin) [inline app/__tests__/scores.test.js:21]: You can keep these though :)

File: README.md [hunk 4]
```diff
@@ -9,3 +9,17 @@
 - [Theo]()
 
 - Coach : [Charlie](https://github.com/Charlie-robin)
+
```
Comment (Charlie-robin) [inline README.md:12]: Can you add:
- A sub title `### Name Of Job` 
  - The name of the Job you created.
- Where in the config do you need to go to update the branch?

File: README.md [hunk 5]
```diff
@@ -9,3 +9,17 @@
 - [Theo]()
```
Comment (Charlie-robin) [inline README.md:9]: Can you put the collaborator's section at the bottom, and give you section a heading `## CI/CD & Jenkins`

Can you also separate the sections with lines? `---`

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (Adaz99) [reply .github/workflows/main.yml]: done

File: app/__tests__/scores.test.js (hunk 2 above)
Comment (Adaz99) [reply app/__tests__/scores.test.js]: done

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**